src/extensions/default/JavaScriptRefactoring/ExtractToVariable.js

```diff
@@ -63,6 +63,17 @@ define(function(require, exports, module) {
             expns[i].start  = doc.adjustPosForChange(expns[i].start, varDeclaration.split("\n"), insertStartPos, insertStartPos);
             expns[i].end    = doc.adjustPosForChange(expns[i].end, varDeclaration.split("\n"), insertStartPos, insertStartPos);
 
+            /* If there are multiple expressions . then second Expression onward
```

Comment:
Instead of adjusting the position for every replace, can you explore replacing backwards, so that the position need not be adjusted for every change. You can refer (and maybe use) [this function](https://github.com/adobe/brackets/blob/master/src/document/Document.js#L631)

Reply:
@boopeshmahendran  We need to highlight the replaced text so replacing backwards, or "MultipleEdits" function won't help here